src/UnitTests/MappingInheritance/IncludedBaseMappingShouldInheritBaseMappings.cs

```diff
@@ -320,6 +341,23 @@ public void include_should_apply_null_substitute()
 
         dest.BaseString.ShouldBe("12345");
     }
+
+    [Fact]
+    public void included_mapping_should_inherit_base_constructor_mappings_should_not_throw()
+    {
+        var config = new MapperConfiguration(cfg =>
+        {
+            cfg.ShouldUseConstructor = constructor => constructor.IsPublic;
+            cfg.CreateMap<RecordOtherObject, RecordObject>()
+                .ForCtorParam(nameof(RecordObject.DifferentBaseString), m => m.MapFrom(s => s.BaseString))
+                .Include<RecordOtherSubObject, RecordSubObject>()
+                .Include<RecordOtherSubObject, RecordSubObjectWithExtraParam>();
```

Comment:
The last `Include` should be in the map below. See https://docs.automapper.org/en/latest/Mapping-inheritance.html.